graylog-storage-elasticsearch6/src/test/java/org/graylog/storage/elasticsearch6/testing/FixtureImporterES6.java

```diff
@@ -144,4 +159,16 @@ private void importNode(JsonNode root) throws IOException {
             throw new IllegalStateException(sb.toString());
         }
     }
+
+    private boolean indexExists(String indexName) {
+        final IndicesExists.Builder request = new IndicesExists.Builder(indexName);
+        final JestResult result = JestUtils.execute(jestClient, request.build(), () -> "Unable to check if index exists: " + indexName);
+
+        return result.isSucceeded();
```

Comment:
`JestUtils#execute` will always throw an exception, if `result.isSucceeded() == false`

```suggestion
         try {
            final JestResult result = jestClient.execute(request.build());
            return result.isSucceeded();
        } catch (IOException e) {
            throw new RuntimeException("Unable to check if index exists: " + indexName, e);
        }
```

Reply:
:heavy_check_mark: